openhcl/openhcl_boot/src/main.rs

```diff
@@ -874,6 +874,10 @@ mod test {
 
     // ensure we can boot with a _lot_ of vcpus
     #[test]
+    #[cfg_attr(
+        target_arch = "aarch64",
+        ignore = "TODO: investigate why this doesn't always work on ARM"
```

Comment:
can you file a github issue on this one - i wonder if there's an actual bug here or not...

Reply:
Ok. It worked in WSL on my arm laptop when I doubled the buffer, but it won't work in CI even with a much bigger buffer. Might be something windows arm specific.

Reply:
https://github.com/microsoft/openvmm/issues/282